modules/run_docker.nf

```diff
@@ -14,7 +14,7 @@ process RUN_DOCKER {
     val ready
 
     output:
-    tuple val(submission_id), path('predictions.csv')
+    tuple val(submission_id), path('predictions.{csv,zip}')
```

Comment:
We went with the route of having all the predictions files for each submission stored under a `predictions` folder. Since there can be multiple predictions files for a single submission, users may be required to zip their predictions up. This change allows the handling of zip files in addition to single csv files.

This also means the `validate.py` needs to handle zip files too.

Reply:
Right now I'm thinking it's best to explicitly call for the extension types so that we know what kind of data files our code has been adapted to support. Also to prevent errors: I can see the possibility of someone accidentally uploading a model that outputs a `.png` file or something, and it breaking `create_folders.py`, because the code doesn't currently support that extension.

Reply:
Well in that case it would still fail in this workflow process, just when it tries to find the output files rather than the python script so I don't think that makes much of a difference. But, we can keep it this way for now.

Reply:
You're right, it would fail later down the line, but with a vague error message as opposed to "this file type is incorrect" which makes troubleshooting harder. I think regardless of where (whether in RUN_DOCKER or UPDATE_FOLDERS etc), it's a good idea to make explicit what file types the workflow can handle for the sake of troubleshooting and documentation.

Reply:
Consider https://docs.python.org/3/library/shutil.html#shutil.unpack_archive - Or another library that may do this for you. The idea is that you shouldn't need to care too much about what the archive is.